Classes/Services/Email/Notifier.php

```diff
@@ -149,6 +149,55 @@ public function sendIngestNotification(\EWW\Dpf\Domain\Model\Document $document)
 
     }
 
+    public function sendRegisterNotification(\EWW\Dpf\Domain\Model\Document $document)
+    {
+
+        try {
+            $client = $this->clientRepository->findAll()->current();
+            $clientAdminEmail = $client->getAdminEmail();
+            $mods = new \EWW\Dpf\Helper\Mods($document->getXmlData());
+            $slub = new \EWW\Dpf\Helper\Slub($document->getSlubInfoData());
+            $submitterEmail = $slub->getSubmitterEmail();
+            $documentType = $this->documentTypeRepository->findOneByUid($document->getDocumentType());
+            $authors = $document->getAuthors();
+
+            $args['###CLIENT###'] = $client->getClient();
+            $args['###PROCESS_NUMBER###'] = $document->getProcessNumber();
+            $args['###DOCUMENT_TYPE###'] = $documentType->getDisplayName();
+            $args['###TITLE###'] = $document->getTitle();
+            $args['###AUTHOR###'] = array_shift($authors);
+
+            $args['###SUBMITTER_NAME###'] = $slub->getSubmitterName();
+            $args['###SUBMITTER_EMAIL###'] = $submitterEmail; //
+            $args['###SUBMITTER_NOTICE###'] = $slub->getSubmitterNotice();
+
+            $args['###DATE###'] = (new \DateTime)->format("d-m-Y H:i:s");
+            $args['###URN###'] = $mods->getQucosaUrn();
+            $args['###URL###'] = 'http://nbn-resolving.de/' . $mods->getQucosaUrn();
+
+
+            // Notify client admin
+            if ($clientAdminEmail) {
+                $subject = $client->getAdminRegisterDocumentNotificationSubject();
+                $body = $client->getAdminRegisterDocumentNotificationBody();
+                $mailType = 'text/html';
+
+                if (empty($subject)) {
+                    $subject = \TYPO3\CMS\Extbase\Utility\LocalizationUtility::translate('LLL:EXT:dpf/Resources/Private/Language/locallang.xlf:notification.registerDocument.admin.subject', 'dpf');
+                }
+
+                if (empty($body)) {
+                    $body = \TYPO3\CMS\Extbase\Utility\LocalizationUtility::translate('LLL:EXT:dpf/Resources/Private/Language/locallang.xlf:notification.registerDocument.admin.body', 'dpf');
+                    $mailType = 'text/plain';
+                }
+
+                $this->sendMail($clientAdminEmail, $subject, $body, $args, $mailType);
+
+            }
+
+        } catch (\Exception $e) {}
```

Comment:
Empty catch block! Errors should be at least logged for later analysis.

Reply:
The errors are now logged with the typo3 logger.